### PR TITLE
Fix attachment chunking

### PR DESCRIPTION
**Pull request by BalmungSan**
Related #24

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -29,8 +30,9 @@ lazy val root = (project in file("."))
       "org.typelevel"     %% "cats-effect"                % catsEffectVersion,
       "co.fs2"            %% "fs2-core"                   % fs2Version,
       "co.fs2"            %% "fs2-io"                     % fs2Version,
+      "com.chuusai"       %% "shapeless"                  % shapelessVersion,
```
Comment (minosiants) [inline build.sbt:33]: why do we need `shapless` ?

File: src/main/scala/com/minosiants/pencil/Smtp.scala [hunk 2]
```diff
@@ -265,20 +279,22 @@ object Smtp {
...
+          Error.smtpError[F, Unit]("Does not support multipart")
+      }
     }
 
   def lines[F[_]: Applicative](txt: String): Smtp[F, Unit] = Smtp[F] { req =>
     txt
       .grouped(76)
       .toList
-      .traverse { ln =>
+      .traverse_ { ln =>
```
Comment (minosiants) [inline src/main/scala/com/minosiants/pencil/Smtp.scala:296]: 👍

File: build.sbt (hunk 1 above)
Comment (BalmungSan) [reply build.sbt:33]: That is a good question, **sbt-explicit-dependencies** says this project depends directly _(meaning there is a direct import somewhere)_ on **Shapeless**.
Probably there is a derivation somewhere.

Comment (minosiants) [reply build.sbt:33]: ok. weird. I know `scodec` uses it.

**Pull request merged**